modules/security-lti/src/main/java/org/opencastproject/security/lti/LtiLaunchAuthenticationHandler.java

```diff
@@ -160,8 +162,8 @@
   /** concurrent attemtps */
   private Map<String, Boolean> activePersistenceTransactions = new ConcurrentHashMap<>(128);
 
-  /** Determines whether a JpaUserReference should be created on lti login */
-  private boolean createJpaUserReference = true;
+  /** Contains the LTI roles where a JpaUserReference shall be created. The default is 'instructor'. */
```

Comment:
```suggestion
  /** LTI roles a user must have so a JpaUserReference is created. */
```

Reply:
The key "createJpaUserReference" is included again (default true).